src/pages/en/guides/content-collections.mdx

```diff
@@ -16,166 +16,196 @@ import TypeScriptSettingTabs from '~/components/tabs/TypeScriptSettingTabs.astro
+
+```json title="tsconfig.json" ins={5}
+{
+  // Note: No change needed if you use "astro/tsconfigs/strict" or "astro/tsconfigs/strictest"
+  "extends": "astro/tsconfigs/base",
+  "compilerOptions": {
+    "strictNullChecks": true
+  }
+}
+```
+
+### Defining a collection schema
 
-To configure schemas, create a `src/content/config.ts` file (`.js` and `.mjs` extensions are also supported). This file should:
+Schemas enforce consistent frontmatter within a collection. A schema **guarantees** that your frontmatter exists in a predictable form when you need to reference or query it. If any file violates its collection schema, Astro will provide a helpful error to let you know.
 
-1. Import the `defineCollection` and `z` utilities from `astro:content`. 
-2. Define a `schema` for each collection.
-2. Export a single `collections` object, with each object key corresponding to the collection's folder name.
+Schemas also power Astro's automatic TypeScript typings for your content. When you define a schema for your collection, Astro will automatically generate and apply a TypeScript interface to it. The result is full TypeScript support when you query your collection, including proprety auto-completion and type-checking. 
 
-For example, say you maintain two collections: one for release announcements and one for blog content. Your entries at `src/content/releases/` should include a `title` and `version`.  Your `src/content/engineering-blog/` collection entries should have a `title`, list of `tags`, and an optional `image` URL. 
+To create your first content schema, create a `src/content/config.ts` file if one does not already exist (`.js` and `.mjs` extensions are also supported). This file should:
 
-You can specify the expected shape of your frontmatter with the `schema` field of `defineCollection`:
+1. Import the proper utilities from `astro:content`. 
+2. Define each collection that you'd like to validate with a schema.
+3. Export a single `collections` object to register your collections.
 
 ```ts
 // src/content/config.ts
+// 1. Import utilities from `astro:content`
 import { z, defineCollection } from 'astro:content';
-
-const releases = defineCollection({
-  schema: z.object({
-    title: z.string(),
-    version: z.number(),
-  }),
-});
-
-const engineeringBlog = defineCollection({
+// 2. Define a schema for each collection you'd like to validate.
+const blogCollection = defineCollection({
   schema: z.object({
     title: z.string(),
     tags: z.array(z.string()),
     image: z.string().optional(),
   }),
 });
+// 3. Export a single `collections` object to register your collection(s)
+export const collections = {
+  'blog': blogCollection,
+};
+```
+
+### Defining multiple collections
 
+You can define as many collections in your project as you would like. As long as they are exported from the `src/content/config.ts` file from inside the `collections` object, they will be loaded by Astro.
```

Comment:
```suggestion
You can use `defineCollection()` as many times as you want to create multiple schemas.  All collections must be exported from inside the single `collections` object.
```
I think this might be clearer, more explicitly hitting home "defineCollections many times, export a collections object once."